index.html

```diff
@@ -28,19 +28,22 @@
   </header>
 
   <section class="search-section">
-    <input type="text" placeholder="Search software jobs" />
-    <button class="search-button">
-      <span class="fa fa-search"></span>
-    </button>
+    <form action="#">
+      <label for="search">Search Jobs</label>
+      <input type="text" id="search" placeholder="Search for a software job" />
```

Comment:
In other words, the `<label>` describes what the input does. The `placeholder` tells them what to do